### PR TITLE
Cannonball explosions

### PR DESCRIPTION
**Pull request by carlfjones**
Cannon explosions done,
Ghosts moving done
Everyone having a great time done

### CODE REVIEW

File: unity_game/Assets/Scripts/BigExplosionEffect.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BigExplosionEffect : MonoBehaviour
```
Comment (Danielandro) [inline unity_game/Assets/Scripts/BigExplosionEffect.cs:5]: Finally, we have explosions

File: unity_game/Assets/Scripts/CannonBallOne.cs [hunk 2]
```diff
@@ -5,9 +5,12 @@
 public class CannonBallOne : MonoBehaviour
 {
     // Start is called before the first frame update
+    public ParticleSystem explosion;
     [SerializeField]
     GameObject arrow;
```
Comment (Danielandro) [inline unity_game/Assets/Scripts/CannonBallOne.cs:10]: Need to change the name of this to cannonball

File: unity_game/Assets/Scripts/Dragon.cs [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 public class Dragon : MonoBehaviour
 {
-    public float upForce = 250f;
+    public float upForce = 200f;
```
Comment (Danielandro) [inline unity_game/Assets/Scripts/Dragon.cs:7]: Might need to play with this to figure out a good balance for agility

File: unity_game/Assets/Scripts/FloatBehaviour2.cs [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+    void Start()
+    {
+        this.originalY = this.transform.position.y;
+    }
+
+    void Update()
+    {
+        transform.position = new Vector3(transform.position.x,
+            originalY + ((float)Math.Sin(Time.time + 1) * floatStrength),
```
Comment (Danielandro) [inline unity_game/Assets/Scripts/FloatBehaviour2.cs:21]: Always add one!

File: unity_game/Assets/Scripts/GhostPool.cs [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    void Update()
+    {
+        StartCoroutine(Example());
+
+    }
+
+    IEnumerator Example()
+    {
+      timeSinceLastSpawned += Time.deltaTime;
+        yield return new WaitForSeconds(3);
```
Comment (Danielandro) [inline unity_game/Assets/Scripts/GhostPool.cs:40]: Glad we got the co-routine working

**Review by binaryred01 — APPROVED** [review]
Man like Carl. OG.

**Review by Danielandro — APPROVED** [review]
Looking really good

**Pull request merged**